src/inc/gcinfoencoder.h

```diff
@@ -491,7 +496,11 @@ class GcInfoEncoder
     GcInfoArrayList<LifetimeTransition, 64> m_LifetimeTransitions;
 
     bool   m_IsVarArg;
+#if defined(_TARGET_AMD64_)
     bool   m_WantsReportOnlyLeaf;
+#elif defined(_TARGET_ARM_) || defined(_TARGET_ARM64_)
+    bool   m_HasTailCalls;
+#endif // _TARGET_AMD64_
```

Comment:
wrong comment for `#endif`, it should be `#endif defined(_TARGET_ARM_) || defined(_TARGET_ARM64_) ` according to https://github.com/dotnet/coreclr/blob/master/Documentation/coding-guidelines/clr-jit-coding-conventions.md#75-commenting-ifdefs, but it is not jit sources, so maybe we do not care.

Reply:
It seems that the coding convention doc actually doesn't cover the #elif pattern and in coreclr runtime, we use the pattern I've used (the comment on #endif matching the #if and not the #elif comment)